app/components/SideBar.js

```diff
@@ -21,12 +21,10 @@ export default class SideBar extends Component {
         this.setState({hidden: true});
     }
 
-    buildUserPlaylist(userPlaylist){
+    buildUserPlaylist(userList){
     	let fields = [];
-    	for(let item in userPlaylist){
-			if(item !== 'favourites' && item !== 'history'){
-    			fields.push(<li><Link to={"/userList/"+item} activeClassName="active">{item}</Link></li>)
-			}
+    	for(let item in userList){
```

Comment:
ad fav check back. its still inside userlist. We'll be showing fav twice in this case.

Reply:
i will change it, forgot to make those changes